client/src/main/java/com/sap/mobile/services/client/push/PushRegistration.java

```diff
@@ -0,0 +1,19 @@
+package com.sap.mobile.services.client.push;
+
+public interface PushRegistration {
+	String getId();
+
+	String getDeviceId();
```

Comment:
/** The device unique ID, used when sending a notification to a specific device @link .... **/